tethys_cli/proxyapps_commands.py

```diff
@@ -0,0 +1,208 @@
+            print(f"  {proxy_app.name}")
+
+        session.close()
+
+
+def get_engine():
+    engine = None
+
+    tethys_settings = read_settings()
+    database_settings = tethys_settings.get("DATABASES", "")
+
+    if database_settings == "":
+        write_error("No database settings defined in the portal_config.yml file")
+        return engine
+
+    database_default_settings = database_settings.get("default", "")
+
+    if database_default_settings == "":
+        write_error("No default database defined in the portal_config.yml file")
+        return engine
+
+    database_engine = database_default_settings.get("ENGINE", "")
+    database_name = database_default_settings.get("NAME", "")
+    database_host = database_default_settings.get("HOST", "")
+    database_password = database_default_settings.get("PASSWORD", "")
+    database_port = database_default_settings.get("PORT", "")
+    database_user = database_default_settings.get("USER", "")
+    try:
+        if database_engine == "django.db.backends.postgresql":
+            postgres_uri = f"postgresql://{database_user}:{database_password}@{database_host}:{database_port}/{database_name}"
+        else:
+            postgres_uri = f"sqlite:///{database_name}"
+        engine = create_engine(postgres_uri, pool_pre_ping=True)
+
+    except Exception:
+        write_error("Error when connecting to the database")
+
+    return engine
+
+
+def proxyapp_command(args):
+    if args.list:
+        list_proxyapps()
+    elif args.add:
+        add_proxyapp(args.add)
+    elif args.update:
+        update_proxyapp(args.update)
+
+
+def update_proxyapp(args):
+    app_name = args[0] if len(args) > 0 else None
+    app_key = args[1] if len(args) > 1 else None
+    app_value = args[2] if len(args) > 2 else None
+
+    if app_name is None:
+        write_error("proxy_app_name cannot be empty")
+        return
+    if app_key is None:
+        write_error("proxy_app_key cannot be empty")
+        return
+    if app_value is None:
+        write_error("proxy_app_value cannot be empty")
+        return
+
+    engine = get_engine()
+
+    if engine:
+        Session = sessionmaker(bind=engine)
+        session = Session()
+
+        Base = automap_base()
+        Base.prepare(engine, reflect=True)
+
+        ProxyAppsClass = Base.classes.tethys_apps_proxyapp
+
+        proxy_app = (
+            session.query(ProxyAppsClass)
+            .filter(ProxyAppsClass.name == app_name)
+            .first()
+        )
+        if proxy_app:
+            if not hasattr(proxy_app, app_key):
+                write_error(
+                    f"Attribute {app_key} does not exists in Proxy app {app_name}"
+                )
+                return
+            setattr(proxy_app, app_key, app_value)
+            session.commit()
+            write_success(f"Proxy app {app_name} was updated")
+        else:
+            write_error(f"Proxy app {app_name} does not exits")
+        session.close()
```

Comment:
Use the Django ORM and the `ProxyApp` class to update proxy apps in the database.